staging/src/k8s.io/code-generator/kube_codegen.sh

```diff
@@ -550,11 +559,12 @@ function kube::codegen::gen_client() {
             group_versions+=("${leaf2}/${leaf}")
         fi
     done < <(
-        ( kube::codegen::internal::git_grep -l \
+        ( kube::codegen::internal::git_grep -l --null \
             -e '+genclient' \
-            ":(glob)${in_root}"/'**/types.go' \
+            ":(glob)${in_root}${one_input_api}"/'**/*.go' \
```

Comment:
shouldn't we not at least exclude generated files? This feels brittle because self-referential, with the risk that clean generation diverges from generation with generated files.